move_together_app/lib/Home/home_screen.dart

```diff
@@ -63,14 +64,18 @@ class _HomeScreenState extends State<HomeScreen> {
                     child: PageView(
                     onPageChanged: changePage,
                     children: state.trips.map((trip) {
+                      var imageUrlToUse =  "https://upload.wikimedia.org/wikipedia/commons/thumb/a/a8/Tour_Eiffel_Wikimedia_Commons.jpg/260px-Tour_Eiffel_Wikimedia_Commons.jpg";
+                      if( trip.photoUrl != null && trip.photoUrl != ""){
+                           imageUrlToUse = "${dotenv.env['API_ADDRESS']}trips/${trip.id}/banner/download";
+                      }
                       return TripCard(
                         onTap: () async {
                           await context.pushNamed('trip', pathParameters: {'tripId': trip.id.toString()});
                           context.read<HomeBloc>().add(HomeDataFetch());
                         },
                         onLeave: () => context.read<HomeBloc>().add(HomeDataLeaveTrip(trip)),
                         onDelete: () => context.read<HomeBloc>().add(HomeDataDeleteTrip(trip)),
-                        imageUrl: "https://upload.wikimedia.org/wikipedia/commons/thumb/a/a8/Tour_Eiffel_Wikimedia_Commons.jpg/260px-Tour_Eiffel_Wikimedia_Commons.jpg",
+                        imageUrl: imageUrlToUse,
```

Comment:
```suggestion
                        imageUrl: trip.photoUrl,
```

Comme maintenant t'as un default dans le model, tu peux directement lui passer `photoUrl`. La condition se fait dans le model

Reply:
au final j'enleve la valeur "photoUrl" du model vu qu'elle n'est plus renvoyé coté backend